docs/api-reference-1/wordpress-source.md

```diff
@@ -341,11 +341,11 @@ Returns an object that gives you info about the type of that link and related en
 {% hint style="info" %}
 `(link: string) => object`
 
-* **Parameters**
-  * **`link`**: `string` Link representing a REST API endpoint or custom handler
-* **Return value**
-  * `object` Info about the type of data represented in the URL
-{% endhint %}
+- **Parameters**
+  - **`link`**: `string` Link that points to a page in the Frontity site.
```

Comment:
Would it be better to say here: "that points to a post or page in the Frontity site."?

Reply:
Uhm, it's a bit confusing, I guess. I meant _any page_ in the Frontity site. Not just posts or pages but also archives, the homepage, etc.

Do you know a better name for that? Page was the best that came to my mind. :see_no_evil: